lib/improver/tests/utilities/cube_manipulation/test_merge_cubes.py

```diff
@@ -120,18 +137,71 @@ def test_lagged_ukv(self):
     def test_multi_model(self):
         """Test Multi models merge OK"""
         cubes = iris.cube.CubeList([self.cube, self.cube_ukv])
-        result = merge_cubes(cubes)
+        result = merge_cubes(cubes, "mosg__model_configuration")
         self.assertIsInstance(result, Cube)
         self.assertArrayAlmostEqual(
-            result.coord("model_realization").points, [3000., 4000.,
-                                                       4001., 4002.])
+            result.coord("model_realization").points, [0., 1.,
+                                                       2., 1000.])
```

Comment:
Do we have a test for this simple multi_model case in which we don't specify the model_id_attr, that is to say it goes in as None? What happens in that case? I'd like to see a test for this if it doesn't exist.

Reply:
I've added a test to show that this results in an iris MergeError. This is caused by the lack of a model_id coordinate, this not being created in the absence of the model_id_attr as this is now effectively a flag for model blending.